starknet-replay/src/histogram/plot.rs

```diff
@@ -0,0 +1,102 @@
+//! This module uses the library `plotter` to render and export the histogram.
+//!
+//! Some settings are hardcoded like:
+//!  - Size and font of labels
+//!  - Colours
+//!  - Export format
```

Comment:
Obvious to anybody who is reading the code, and unnecessary for the users of the API.